ccd-definition/CaseEvent/CaseEvent.json

```diff
@@ -682,5 +682,18 @@
     "PreConditionState(s)": "Gatekeeping",
     "PostConditionState": "PREPARE_FOR_HEARING",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "draftCMO",
+    "Name": "Draft CMO",
```

Comment:
is this name correct? why not Draft Case Management Order

Reply:
I checked with Moray and they want to show as 'Draft CMO' in event list